### ISSUE TITLE
 How can I import system and region maps?

### ISSUE THREAD

**Issue by HMM18** [goryn-clade/pathfinder#234]
I want to import maps instet to add manualy or maping the system, its posible?

**Comment by TyrHeimdalEVE** [comment]
Yes, you can use the import/export functionality for maps. Please ask on the Slack channel for things like this as it's not really an issue with Pathfinder.
